presto-hudi/src/main/java/com/facebook/presto/hudi/HudiMetadata.java

```diff
@@ -245,7 +246,7 @@ static List<HudiColumnHandle> fromPartitionColumns(List<Column> partitionColumns
         return builder.build();
     }
 
-    static List<HudiColumnHandle> fromDataColumns(List<Column> dataColumns)
+    public static List<HudiColumnHandle> fromDataColumns(List<Column> dataColumns)
```

Comment:
Why the scope change? If it's only being used inside `presto/hudi/split` package then let's move there without changing the scope.